src/main/java/com/google/cloud/genomics/utils/grpc/Channels.java

```diff
@@ -98,7 +98,6 @@ private static Channel getGenomicsChannel() throws SSLException {
     
     Channel channel = NettyChannelBuilder.forAddress("genomics.googleapis.com", 443)
         .negotiationType(NegotiationType.TLS)
-        .streamWindowSize(1000000)
```

Comment:
@ssgross   There is a different method to set that value now.  See https://github.com/grpc/grpc-java/blob/master/netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java#L126

But since the default is 1048576, which is roughly equivalent to what was in the file, I took it out.  See https://github.com/grpc/grpc-java/blob/master/netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java#L54

Let me know if you think we should continue to set it explicitly.